kano_greeter/last_user.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import os
+
+LAST_USER_PATH = '/var/lib/kano-greeter'
```

Comment:
@hzulla I've just tried this and I get a permission error when attempting to write to `/var/lib/kano-greeter`. The greeter is not run as `root` but by a restricted `lightdm` user whose home directory is `/var/lib/lightdm`. The easiest solution might be to just change `LAST_USER_PATH` to be `/var/lib/lightdm/kano-greeter` or `/var/lib/lightdm/.cache/kano-greeter` (as you suggested earlier).